src/components/prescription-details.component.tsx

```diff
@@ -0,0 +1,47 @@
+import { DataTableSkeleton, Tile } from "carbon-components-react";
+import React from "react";
+import styles from "./prescription-details.scss";
+import { WarningFilled24 } from "@carbon/icons-react";
+import { useOrderDetails } from "../medication-request/medication-request.resource";
+import { useTranslation } from "react-i18next";
+import MedicationCard from "./medication-card.component";
+
+const PrescriptionDetails: React.FC<{ encounterUuid: string }> = ({
+  encounterUuid,
+}) => {
+  const { t } = useTranslation();
+  const { medications, isError, isLoading } = useOrderDetails(encounterUuid);
+
+  return (
+    <div className={styles.prescriptionContainer}>
+      <Tile className={styles.allergiesTile}>
+        <div className={styles.allergesContent}>
+          <div>
+            <WarningFilled24 className={styles.allergiesIcon} />
+            <p>
+              <b>3 allergies</b> Penicillin, Naproxen sodium, Ibuprofen
```

Comment:
We need to be careful about inserting dummy data into these components, especially data like this.

Reply:
Understood. Will rectify this in subsequent PRs